### PR TITLE
fix: resolved-workspace-sorting[#1923]

### PR DESCRIPTION
**Pull request by aakashreddy-p**
## Description

I have applied sorting based on the updated time[#1923].

Please do not leave this blank 

## What type of PR is this? (check all applicable)
- [x] 🐛 Bug Fix

## Have you tested locally?
- [x] 👍 yes

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by aakashreddy-p** [comment]
> Please attach the issue link - https://github.com/actions/runner-images/issues/10636

File: packages/@sparrow-teams/src/features/team-explorer/components/workspace-list-view/WorkspaceListView.svelte [hunk 1]
```diff
@@ -66,7 +66,7 @@
               .filter((item) => item.name
                   .toLowerCase()
                   .startsWith(filterText.toLowerCase()))
-              .sort((a, b) => a.name.localeCompare(b.name))
+              .sort((a, b) => new Date(b._data.updatedAt).getTime() - new Date(a._data.updatedAt).getTime())
               .slice((currPage - 1) * workspacePerPage, currPage * workspacePerPage) as list, index}
               <Rows
                 {onAddMember}
```
Comment (itsmdasifraza) [inline packages/@sparrow-teams/src/features/team-explorer/components/workspace-list-view/WorkspaceListView.svelte:72]: Could you implement this for the grid view as well?

**Review by itsmdasifraza — CHANGES_REQUESTED** [review]

File: packages/@sparrow-teams/src/features/team-explorer/components/workspace-list-view/WorkspaceListView.svelte (hunk 1 above)
Comment (aakashreddy-p) [reply packages/@sparrow-teams/src/features/team-explorer/components/workspace-list-view/WorkspaceListView.svelte:72]: sure I will implement for it as well.

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**